### DISCUSSION TITLE
BEMT disabled for negative inflow in OpenFast v2.5 and later

### DISCUSSION THREAD

**Discussion by pelljam** [OpenFAST/openfast#940] [Q&A] [answered]
Hi @jjonkman (and others),

I’ve been trying to understand a change in the behaviour I am seeing in an OpenFast model between versions. I think the main difference comes about because after v2.5 the BEM is turned off (smoothly with TSR) when the TSR falls below 2.

This happens in `check_turnOffBEMT` by testing` u%TSR < BEMT_upperBoundTSR` and then blending the BEMT solution with the non-BEMT/geometric one.

This also means that the BEMT will be disabled for negative inflow onto the rotor, i.e. if the relative inflow velocity used in the calculation of the TSR is negative (well, I guess, unless the rotor is spinning backwards). 

For example, if you had a conventionally rotating turbine, which was spinning quickly, with a large positive TSR, but at some point, the relative wind direction changed and came from behind (e.g. due to a FOWT pitching backwards) then the induction would be zero during this phase of negative inflow. Also, I guess, as the inflow velocity goes from positive to negative, the induction will change discontinuously. Because, whilst the BEMT and non-BEMT solutions are blended smoothly on TSR, they will not be blended smoothly on inflow velocity going from positive to negative, because the TSR->inf as V->0.

I just wanted to double check that this is the intended behaviour, i.e. there is no induction for negative TSR values? Apologies if I have got the wrong end of the stick!

Thanks,
James

**Comment by bjonkman** [comment, marked as answer]
@pelljam, Thanks for the question. It seems like we should have implemented that part of the code using the `abs(TSR)` so that BEM is turned off only when TSR is near 0, not for all negative values. I will update the code and make a pull request.

**Reply by pelljam** [reply]
@bjonkman thanks for the quick reply and clarification. Yes, that was my suspicion too.

Looks like this should probably have gone in as an _issue_ then. Seems I am incapable of putting these things in the right place. Sorry!

James
